fastlane/Constants/Constant.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  Constant.swift
+//  FastlaneRunner
+//
+//  Created by Su Ho on 22/09/2022.
+//  Copyright © 2022 Joshua Liebowitz. All rights reserved.
```

Comment:
I think we should modify the Copyright to Nimble when we are editing these files for the first time.

Reply:
Good catch! Let's me change the organization on the Xcode